code/scripting/api/libs/time_lib.cpp

```diff
@@ -15,5 +15,10 @@ ADE_FUNC(getCurrentTime, l_Time, nullptr, "Gets the current real-time timestamp.
 	return ade_set_args(L, "o", l_Timestamp.Set(timer_get_nanoseconds()));
```

Comment:
I think the documentation should be more explicit for this.  Something like "Gets the current real-time timestamp, not taking into account mission pause or UI use."